src/extlibs/enlsvg/README.md

```diff
@@ -1,7 +1,7 @@
-# ENLSVG-Pathfinding
+# Enlsvg-Pathfinding
 This is an implementation of the Edge N-Level Sparse Visibility Graph algorithm for extremely fast optimal Any-Angle Pathfinding on grid maps. (on the order of a millisecond per path computation on 6000x6000 grids)
 
-The ENLSVG algorithm requires a pre-processing step, which may take some time, depending on the size of the grid map.
+The Enlsvg algorithm requires a pre-processing step, which may take some time, depending on the size of the grid map.
```

Comment:
I think we can leave it as ENLSVG (all caps) in the readme? line 1 as well

Reply:
Oh, thanks for pointing that out! I created a new pull request #2755 to fix that.